.github/workflows/causal_lm_cpp.yml

```diff
@@ -809,6 +809,7 @@ jobs:
         run: |
           source ./ov/setupvars.sh
           timeout 60s ./build/samples/cpp/text_generation/benchmark_genai -m ./TinyLlama-1.1B-Chat-v1.0/ -p "Why is the sun yellow?" --nw 2 -n 3 --mt 50 -d CPU
+          timeout 60s  python ./samples/python/text_generation/benchmark_genai.py -m ./TinyLlama-1.1B-Chat-v1.0/ -p "Why is the sun yellow?" -nw 2 -n 3 -mt 50 -d CPU
```

Comment:
```suggestion
          timeout 60s  python ./samples/python/text_generation/benchmark_genai.py -m ./TinyLlama-1.1B-Chat-v1.0/ -p "Why is the sun yellow?" --nw 2 -n 3 -mt 50 -d CPU
```

Reply:
No :)
For python -nw (-mt) and for cpp --nw (--nw). I already updated docs appropriately

Reply:
https://github.com/openvinotoolkit/openvino.genai/actions/runs/13370505008/job/37337756523?pr=1747